samples-and-tests/just-test-cases/test/routing.test.html

```diff
@@ -185,20 +185,19 @@
     assertTextPresent('21: http://localhost:9003/hello%20world')
     assertTextPresent('22: http://localhost:9003/st%C3%A9phane?name=nicola%C3%AFs')
     assertTextPresent('23: http://localhost:9003/hello%20world?name=nicola%C3%AFs')
-
+    
     open('/client')
     assertTextPresent('localhost')
-
-
+    
     // jsRoute tag
     open('@{Application.jsRoute()}')
     assertText('a-url', '/')
     assertText('a-method', 'GET')
     assertText('b-url', '/sayHello')
     assertText('b-method', 'PUT')
     assertText('c', '/ressource/foo')
-
-    open('@{Application.jsRouteError()}')
-    assertTextPresent('Wrong parameter type')
+    // FIXME: This passes but an error is recorded that stops further testing!
+//    open('@{Application.jsRouteError()}')
+//    assertTextPresent('Wrong parameter type')
```

Comment:
@xael-fry are you ok with this test being commented out? They don't pass currently and I was unable to resolve

Reply:
@tazmaniax yes as you keep it and put some comment to explain, I think we ca merge it